provider/cluster_rosa_classic_resource.go

```diff
@@ -855,59 +811,41 @@ func (r *ClusterRosaClassicResource) Delete(ctx context.Context, request tfsdk.D
 		)
 		return
 	}
-
-	// Wait till the cluster has been effectively deleted:
-	if !state.WaitWithTimeout.Unknown && !state.WaitWithTimeout.Null {
-		if state.WaitWithTimeout.Value <= 0 {
-			response.Diagnostics.AddWarning(nonPositiveTimeoutSummary, fmt.Sprintf(nonPositiveTimeoutFormat, state.ID.Value))
-		} else {
-			isNotFound, err := r.waitTillClusterIsNotFoundWithTimeout(ctx, state.WaitWithTimeout.Value, resource, r.logger)
-			if err != nil {
-				response.Diagnostics.AddError(
-					"Can't poll cluster state",
-					fmt.Sprintf(
-						"Can't poll state of cluster with identifier '%s': %v",
-						state.ID.Value, err,
-					),
-				)
-				return
+	if !state.DisableWaitingInDestroy.Unknown && !state.DisableWaitingInDestroy.Null && state.DisableWaitingInDestroy.Value {
+		r.logger.Info(ctx, "Disable waiting")
```

Comment:
This log is not clear, write something like "Wait for destroy complete is disabled"